web/src/lib/components/memory-page/memory-viewer.svelte

```diff
@@ -246,29 +239,33 @@
 						<div class="absolute h-full flex justify-between w-full">
 							<div class="flex h-full flex-col place-content-center place-items-center ml-4">
 								<div class="inline-block">
-									<CircleIconButton
-										logo={ChevronLeft}
-										backgroundColor="#202123"
-										on:click={toPreviousCurrentAsset}
-									/>
+									{#if previousMemory || assetIndex > 0}
+										<CircleIconButton
+											logo={ChevronLeft}
+											backgroundColor="#202123"
+											on:click={toPreviousAsset}
```

Comment:
Usually click methods use `handlePreviousAsset` or `onPreviousAsset` naming convention, depending if it is being handled or propagated.

Reply:
Makes sense, this is just what the original code was.